### ISSUE TITLE
Error: Missing getServerSnapshot on latest @nhost/nextjs 1.2.3 @nhost/react 0.7.3

### ISSUE THREAD

**Issue by alveshelio** [nhost/nhost#537]
After upgrading to latest version of nhost `@nhost/nextjs 1.2.3` and `@nhost/react 0.7.3` I've started getting this error

Error: Missing getServerSnapshot, which is required for server-rendered content. Will revert to client rendering.

I though that this was an issue with a library I had installed `zustand` according to this issue: https://github.com/pmndrs/zustand/issues/886, however,  I was not getting this error before. 

I've done a couple of tests, I've  updated `zustand` to version `4.0.0-rc1` with `@nhost/nextjs 1.2.3` and `@nhost/react 0.7.3` and this still persisted. When I downgrade to `@nhost/nextjs 1.2.2` and `@nhost/react 0.7.2` the error goes away.

**Comment by szilarddoro** [comment]
Hey Helio, it seems that it has been fixed in `zustand@4.0.0-beta.3`. Are you sure that this is an issue with our packages? We are not using `zustand` and this error is most likely coming from their library.

**Comment by alveshelio** [comment]
HI @szilarddoro,

Indeed, it has been solved in `zustand@4.0.0-beta.3` that's why I think the issue is with Nhost packages. I've tested with a couple of zustand versions and when running with `@nhost/nextjs 1.2.2` and `@nhost/react 0.7.2` I never get the error, the only time I get the error is when I upgrade to `@nhost/nextjs 1.2.3` and `@nhost/react 0.7.3`.

I'm not sure if it's going to be easy to create a repo since this is a private repo. Maybe I can check with @plmercereau like we did last time see if we can find the issue.
